app/assets/javascripts/uploadFiles.js

```diff
@@ -0,0 +1,222 @@
+;(function (global) {
```

Comment:
why this `;` in front?

Reply:
following the same code to expose globals as show-hide-content.js

Reply:
I dont think its technically required in this instance. You would need it if you didnt have the outer brackets e.g.
`;function(global){...}` to stop the JS compiler thinking it's an expression